### PR TITLE
SetOffloadthreshold

### PR DESCRIPTION
**Pull request by ext-devgangavkar**

### CODE REVIEW

**Comment by pgier** [comment]
There are some conflicts, please rebase onto the latest master.

**Comment by pgier** [comment]
Can you squash these commits and add a meaningful commit message?

File: internal/provider/types_streaming_namespace_policies.go [hunk 1]
```diff
@@ -186,6 +189,7 @@ var (
 					policyRetentionSizeInMB:      int64PulsarNamespacePolicyAttribute,
 				},
 			},
+
```
Comment (pgier) [inline internal/provider/types_streaming_namespace_policies.go:192]: Remove this newline

File: internal/provider/types_streaming_namespace_policies.go [hunk 2]
```diff
@@ -335,13 +340,20 @@ func setNamespacePolicies(ctx context.Context, client *pulsaradmin.ClientWithRes
...
+	}
+
 	if policies.InactiveTopicPolicies != nil {
 		resp, err := client.NamespacesSetInactiveTopicPolicies(ctx, tenant, namespace, *policies.InactiveTopicPolicies, requestEditors...)
 		diags.Append(HTTPResponseDiagWarn(resp, err, pulsarNamespacePolicyError(policyInactiveTopicPolicies))...)
 	}
 	if policies.SubscriptionExpirationTimeMinutes != nil {
 		resp, err := client.NamespacesSetSubscriptionExpirationTime(ctx, tenant, namespace, *policies.SubscriptionExpirationTimeMinutes, requestEditors...)
 		diags.Append(HTTPResponseDiagWarn(resp, err, pulsarNamespacePolicyError(policySubscriptionExpirationTimeMinutes))...)
+
```
Comment (pgier) [inline internal/provider/types_streaming_namespace_policies.go:356]: remove this newline

**Comment by ext-devgangavkar** [comment]
> Can you squash these commits and add a meaningful commit message?

Sure

**Comment by pgier** [comment]
Closing in favor of #381

**Pull request closed without merging**